README.md

```diff
@@ -103,4 +104,43 @@ input = wrapField(Input);
   component={this.input}
   className={styles.input}
 />
-```
\ No newline at end of file
+```
+
+### Spinner
+
+The component contain this props:
+* WrappedComponent
+* classNameContainer
+* classNameLoading
+* idLoading
+* colorSpinner
+
+Contain this component:
+* Loading 
+  * className
+  * id
+  * colorSpinner
+
+
+In the example we will use it with the SearchBar component.
+
+```
+src/components/SearchBar
+
+//Import into the component to be used.
+import { withSpinner } from '../Spinner';
+
+//Include in the definition of the component to be used. In this case is SearchBar.
+export default withSpinner({
+  WrappedComponent: SearchBarContainer,
```

Comment:
A nice approach would be 
```js 
export default  withSpinner({idLoading: 1, classNameContainer: styles.spinner})(SearchBarContainer)
```
This is a classic pattern used by many packages. `ẀithSpinner` must return a function that receives a component and returns another one. The param will be like the wrapped component.